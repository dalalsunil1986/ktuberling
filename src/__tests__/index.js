jest.mock('react-dom', () => ({ render: jest.fn() }));
jest.mock('../app', () => 'App');

const ReactDom = require('react-dom');

require('../index');

describe('App', () => {
  it('init', () => {
    expect(ReactDom.render).toHaveBeenCalledTimes(1);
    expect(ReactDom.render.mock.calls).toMatchSnapshot();
  });
});
