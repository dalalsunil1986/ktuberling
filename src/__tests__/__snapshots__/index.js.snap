// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`App init 1`] = `
Array [
  Array [
    <App />,
    <div
      id="app"
    />,
  ],
]
`;
